docker/compose/docker-compose-nfs.yml

```diff
@@ -0,0 +1,91 @@
+#
```

Comment:
Docker compose is just for local development or testing. It all runs on a single node. Someone looking to deploy a scalable cluster would use [docker swarm](https://github.com/pravega/pravega/tree/master/docker/compose/swarm).